utils/generator/templates/scripts/OnHMIStatus_format.java

```diff
@@ -0,0 +1,25 @@
+
+    @Override
```

Comment:
The generator is making the `firsRun` as deprecated for some reason:
```
/**
 * @deprecated
 */
@Deprecated
private Boolean firstRun;
```

Reply:
fixed